e2e/apprunner/apprunner_test.go

```diff
@@ -27,6 +27,7 @@ var _ = Describe("App Runner", Ordered, func() {
 	BeforeAll(func() {
 		_, initErr = cli.Init(&client.InitRequest{
 			AppName:      appName,
+			EnvName:      envName,
```

Comment:
@bvtujo just to doublecheck: we don't make "test" default value for env name and will ask them for the value?

Reply:
that sounds right to me based on https://github.com/aws/copilot-cli/pull/5168!